Sources/Extensions/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -13,13 +13,11 @@ public extension Array where Element: Numeric {
 	
 	/// SwifterSwift: Sum of all elements in array.
 	///
-	///		[1, 2, 3, 4, 5].sum -> 15
+	///		[1, 2, 3, 4, 5].sum() -> 15
 	///
 	/// - Returns: sum of the array's elements.
 	public func sum() -> Element {
-        var total: Element = 0
-        forEach { total += $0 }
-        return total
+        return self.reduce(0 as Element, +)
```

Comment:
We probably want to use `reduce(into:)` because we're dealing with a copy-on-write type